### ISSUE TITLE
build.gradle plugin classpath is specified twice

### ISSUE THREAD

**Issue by jonsmirl** [google-home/sample-app-for-matter-android#24]
```
        classpath "com.google.dagger:hilt-android-gradle-plugin:2.38.1"
        classpath 'com.google.protobuf:protobuf-java:3.4.0'
        classpath 'com.google.dagger:hilt-android-gradle-plugin:2.41'
```
This triggers weird compile errors after a gradle upgrade.

**Comment by pierredelisle** [comment]
Thanks! Fixed in PR #26.

**Comment by pierredelisle** [comment]
Release 1.1.0 now up. I'll upload the APKs sometime tomorrow.
